File: hw/astudillo/U3/Hw19Composite/CompositePython/composite_pattern/model/teller.py
from .employee import Employee

class Teller(Employee):
    def __init__(self, name=None):
        super().__init__(name)
        self.title = "Teller"
    
    def state_name(self, title=None, name=None):
        if title is None and name is None:
            super().state_name(self.title, self.name)
        else:
            super().state_name(title, name)

File: hw/maisincho/u3/HW20Compiste/Composite/src/main/java/ec/edu/espe/composite/model/Teller.java
package ec.edu.espe.composite.model;

import ec.edu.espe.composite.controller.Employee;

/**
 *
 * @author Isaac Maisincho Crafters_Market DCCO ESPE
 */
public class Teller extends Employee {

    public Teller(String aName) {
        this();
        name = aName;
    }

    @Override
    public void stateName() {
        super.stateName();
    }

    public Teller() {
        title = "Teller";
    }

}
